<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serie</class>
 <widget class="QDialog" name="serie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>118</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editar serie</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>SERIE:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="codigolineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>PRÓX.DOC.:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="proxdoclineEdit">
       <property name="inputMask">
        <string>99999999999999; </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>DESCRIPCIÓN:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="descriplineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelarpushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptarpushButton">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelarpushButton</sender>
   <signal>clicked()</signal>
   <receiver>serie</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
